javascript/ql/src/semmle/javascript/frameworks/Electron.qll

```diff
@@ -33,4 +33,38 @@ module Electron {
       this = DataFlow::moduleMember("electron", "BrowserView").getAnInstantiation()
     }
   }
+  
+  /**
+   * A Node.js-style HTTP or HTTPS request made using `electron.net`.
+   */
+  class NetRequest extends NodeJSLib::ClientRequest {
+    NetRequest() {
+      exists(DataFlow::MethodCallNode mcn |
+        mcn = this and
+        DataFlow::moduleMember("electron", "net").flowsTo(mcn.getReceiver()) and
+        mcn.getMethodName() = "request"
```

Comment:
Does `this = DataFlow::moduleMember("electron", "net").getAMemberCall("request")` not do what you want?